component/notification/src/main/java/org/exoplatform/social/notification/impl/SpaceWebNotificationServiceImpl.java

```diff
@@ -208,6 +222,20 @@ public Map<String, Long> countUnreadItemsByApplication(long userIdentityId, long
     return metadataService.countMetadataItemsByMetadataTypeAndAudienceId(METADATA_TYPE_NAME, userIdentityId, spaceId);
   }
 
+  @Override
+  public Map<Long, Long> countUnreadItemsBySpace(String authenticatedUser) {
+    List<Long> spaceIds = new ArrayList<>();
+    IdentityManager identityManager = getIdentityManager();
```

Comment:
you have already an `identityManager` that is injected by constructor

```suggestion
```

Reply:
done!